x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/LifecyclePolicy.java

```diff
@@ -281,41 +259,10 @@ public StepKey getNextValidStep(StepKey stepKey) {
         }
     }
 
-    private StepKey getNextAfterStep(String currentPhaseName) {
```

Comment:
This is the removal of these as a followup to https://github.com/elastic/elasticsearch/pull/33037#discussion_r211906351